lib/config_editor/config_editor_dialog.dart

```diff
@@ -178,6 +188,52 @@ class ConfigEditor extends StatelessWidget {
   }
 }
 
+class _MultiLineTextField extends StatefulWidget {
+  const _MultiLineTextField({required this.initialValue, this.onChanged});
+  final String initialValue;
+  final void Function(String)? onChanged;
+
+  @override
+  State<_MultiLineTextField> createState() => __MultiLineTextFieldState();
+}
+
+class __MultiLineTextFieldState extends State<_MultiLineTextField> {
+  late final ScrollController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = ScrollController();
+  }
```

Comment:
cosmetic nit-pick: we can simplify this to
```dart
final _controller = ScrollController();
```dart

we don't need `initState()` at all when there's no need to access `widget` for initial values, add listeners, or things like that